docs/miscellaneous.rst

```diff
@@ -325,6 +324,7 @@ Global Variables
 - ``assert(bool condition)``: abort execution and revert state changes if condition is ``false`` (use for internal error)
 - ``require(bool condition)``: abort execution and revert state changes if condition is ``false`` (use for malformed input or error in external component)
 - ``revert()``: abort execution and revert state changes
+- ``blockhash(uint blockNumber) returns (bytes32)``: hash of the given block - only works for 256 most recent blocks
```

Comment:
I think all these values should be sorted somehow...

Reply:
Do you have a strong opinion on how they should be sorted? Otherwise I'd probably sort them by variable in alphabetical order.

Reply:
Yes, alphabetically, but that can be a separate PR.